src/lib/features/feature-search/feature.search.e2e.test.ts

```diff
@@ -1425,3 +1425,93 @@ test('should return change request ids per environment', async () => {
         ],
     });
 });
+
+const createReleasePlan = async ({
```

Comment:
we don't have release plan stores in OSS so will move them in another PR instead of manual queries